songpal/method.py

```diff
@@ -152,18 +152,78 @@ async def __call__(self, *args, **kwargs):
 
     @property
     def inputs(self) -> Dict[str, type]:
-        """Input parameters for this method."""
-        return self.signature.input
+        """Input parameters for this method version."""
+        return (
+            self.signatures[self._version].input
+            if self.signatures[self._version].input is not None
+            else ""
+        )
+
+    @property
+    def latest_supported_version(self) -> str:
```

Comment:
```suggestion
    def latest_supported_version(self) -> Optional[str]:
```
as this might be `None`. But is there a chance that there are no versions defined?

Reply:
For my device there are some. No real issue with these, but I wasn't sure about other devices.

```
DEBUG:songpal.device:No supported version for appControl.getMethodTypes, using 1.0
DEBUG:songpal.device:No supported version for appControl.getVersions, using 1.0
DEBUG:songpal.device:No supported version for audio.getMethodTypes, using 1.0
DEBUG:songpal.device:No supported version for audio.getVersions, using 1.0
DEBUG:songpal.device:No supported version for avContent.getMethodTypes, using 1.0
DEBUG:songpal.device:No supported version for avContent.getVersions, using 1.0
DEBUG:songpal.device:No supported version for guide.getMethodTypes, using 1.0
DEBUG:songpal.device:No supported version for guide.getVersions, using 1.0
DEBUG:songpal.device:No supported version for guide.switchNotifications, using 1.0
DEBUG:songpal.device:No supported version for system.getMethodTypes, using 1.0
DEBUG:songpal.device:No supported version for system.getVersions, using 1.0
```